pkg/cloudprovider/types.go

```diff
@@ -74,6 +74,8 @@ type InstanceType interface {
 	// Overhead is the amount of resource overhead expected to be used by kubelet and any other system daemons outside
 	// of Kubernetes.
 	Overhead() v1.ResourceList
+	// MaxVolumes returns the maximum number of volumes that can be mounted to the node.
+	MaxVolumes() int
```

Comment:
Can this be expressed via `Resources()`? 
From a real node:
```
    capacity:
      attachable-volumes-aws-ebs: "25"
      cpu: "8"
      ephemeral-storage: 83873772Ki
      hugepages-1Gi: "0"
      hugepages-2Mi: "0"
      memory: 32521312Ki
      pods: "58"
```

Reply:
See below, that number is incorrect.